### ISSUE TITLE
Problems with training the model

### ISSUE THREAD

**Issue by tholmb** [anshkumar/yolact#7]
Thanks for sharing this great project! I'm new with instance segmentation and would like to learn more about it and manage to train YOLACT tf model. I tried to do it with predefined model parameters but based on evaluation metrics, the training didn't converge. I'm trying to understand the implementation and there are several parts which i don't fully understand. 

1. In the original pyTorch [project](https://github.com/dbolya/yolact) for FPN, Protonet, PredictionHead and Semantic segmentation are used Xavier initialization (glorot_uniform()) but in this project the weights are initialized in different way. Does the training converges faster in this way?

2. MS COCO dataset contains in total 81 classes but in train.sh is defined 90 classes. Is this num_classes parameter somehow related to class IDs?

3. For ResNet50 backbone is reported for qualitative resuls on COCO 28.2 mAP value. Do you have reached that one?

Thanks in advance!

**Comment by anshkumar** [comment]
hi, thanks for your feedback. I'm using the initialization used by tensorflow-object detection api. I'm using class IDs as key so 91 classes (same way tensorflow-object detection uses). The reported accuracy on paper on test-dev (coco) is yet to be tested by me. On validation dataset I'm currently getting 24 MAP. I'll update the accuracy number on test-dev soon, also with the checkpoints.

Also, I've added sigmoid in the losses for faster convergence. Get the latest pull.

**Comment by tholmb** [comment]
Thanks for the reply!  This clarifies a lot of things. 

I also noticed that in the original project batch normalizations of the resnet backbone are freezed after loading imagenet weights. I'm just wondering that is yolact.py line 120 (c3, c4, c5 = self.backbone_resnet(inputs, training=False)) doing the same?

**Comment by anshkumar** [comment]
Yes it does the same thing.
